src/ontology/foodon-edit.owl

```diff
@@ -13172,6 +13181,9 @@ AnnotationAssertion(obo:IAO_0000114 obo:FOODON_00002465 obo:IAO_0000122)
 
 # Class: obo:FOODON_00002470 (food object quality)
 
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_00002470 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00002470 "A physical object quality which can describe an attribute of a food item or related to a food product."@en)
+AnnotationAssertion(obo:IAO_0000117 obo:FOODON_00002470 "0000-0001-8008-8249")
```

Comment:
When manually adding yourself as term editor you need to include the purl prefix. Protege does this for you automatically for new terms but not for ever time you add the annotation.

e.g. use http://orcid.org/0000-0001-8008-8249 like you did in line 66605

```suggestion
AnnotationAssertion(dc:creator obo:FOODON_00002470 <http://orcid.org/0000-0001-8008-8249>)
```

Reply:
I've noticed that outside of manual curation, we primarily use `term editor` [IAO_0000117] (e.g., in ROBOT tables) even for new terms. Going to check with Damion if he'd like us to only use `term editor` moving forward. If so we'll have to disable Protege's automated `dc:creator` annotation upon creation of new terms.

Reply:
Right, forgot we could change it - so let's use `dc:contributor` as default and can discuss this further in Oc/Or to make sure everyone is on board.
![image](https://user-images.githubusercontent.com/48695054/172713147-52d308a9-d432-421e-92e9-a551b77e7f5f.png)